content/docs/setup/kubernetes/minimal-install/index.md

```diff
@@ -15,21 +15,17 @@ Refer to the [prerequisites](/docs/setup/kubernetes/quick-start/#prerequisites)
 
 ## Installation steps
 
-1. If using a Helm version prior to 2.10.0, install Istio's [Custom Resource Definitions](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/#customresourcedefinitions)
-via `kubectl apply`, and wait a few seconds for the CRDs to be committed in the kube-apiserver:
-
-    {{< text bash >}}
-    $ kubectl apply -f install/kubernetes/helm/istio/templates/crds.yaml
-    {{< /text >}}
-
-1. Choose one of the following two
-**mutually exclusive** options described below.
+Choose one of the following two **mutually exclusive** options described below.
 
 ### Option 1: Install with Helm via `helm template`
 
+Choose this option if your cluster doesn't have [Tiller](https://github.com/kubernetes/helm/blob/master/docs/architecture.md#components) deployed and you don't want to install it.
+
 1. Render Istio's core components to a Kubernetes manifest called `istio-minimal.yaml`:
 
     {{< text bash >}}
+    $ cat @install/kubernetes/namespace.yaml@ > $HOME/istio-minimal.yaml
+    $ cat install/kubernetes/helm/istio-init/files/crd-1* >> $HOME/istio-minimal.yaml
     $ helm template install/kubernetes/helm/istio --name istio --namespace istio-system \
       --set security.enabled=false \
       --set ingress.enabled=false \
```

Comment:
No need for this one because disabled by default

Reply:
Nice catch!